crime-commons-mod-schemas/src/main/resources/schemas/passported/apiGetPassportedAssessmentResponse.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiGetPassportedAssessmentResponse.json",
+  "type": "object",
+  "title": "Get Passported Assessment response",
+  "description": "Data contract for getting a Passported Assessment",
+  "allOf": [
+    {
+      "$ref": "common/passportedAssessment.json"
+    },
+    {
+      "properties": {
```

Comment:
Interesting. It's probably up to you here how you go on this one, for IoJ Appeals I added both the id and legacy id as part of the get response, then extended the schema containing the actual IoJ Appeal fields. Reason being is that when creating an appeal (in your case here an assessment) you're not going to have either of these ids, so to me it doesn't make sense to include them in the schema for the object that's being sent when an assessment is created (`passportedAssessment.json` in this PR).